test/parallel/test-http-client-reject-unexpected-agent.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const baseOptions = {
+  method: 'GET',
+  port: undefined,
+  host: '127.0.0.1',
+};
+
+const failingAgentOptions = [
+  true,
+  'agent',
+  {},
+];
+
+const acceptableAgentOptions = [
+  false,
+  undefined,
+  null,
+  new http.Agent(),
+];
+
+
```

Comment:
random whitespace - you have single line between global scope vars everywhere but here